### ISSUE TITLE
Segmentation fault triggered by specific colloid position

### ISSUE THREAD

**Issue by ohenrich** [ludwig-cf/ludwig#70]
We observe a segmentation fault in parallel runs for a specific colloid position. 

The decomposition is 64^3 and the colloid position is 32.5_32.5_32.5. It turns out 32.49999999999999 (14 decimal places) works, but 32.499999999999999 (15 decimal places) doesn't. The reason for this is that the assertion in line #146 in /src/colloids.c fails. 

Attached are input and stdout.

[input.txt](https://github.com/ludwig-cf/ludwig/files/4421893/input.txt)
[stdout.txt](https://github.com/ludwig-cf/ludwig/files/4421894/stdout.txt)

**Comment by kevinstratford** [comment]
Yes; this is a bit of a 'feature'. If you set the position to bang on the
processor decomposition, the colloid can 'disappear' and the assertion
fails...

I'm probably going to mark this as "won't fix" for the time being.

**Comment by ohenrich** [comment]
No problem at all. It happened now for the very first time and placing it 1e-14 away is a good workaround. Maybe it's worth adding a comment to the documentation, but it happens so rarely that I find it hard to convince myself. I'll close this issue now.
